crates/runtime/src/datafusion/query/tracker.rs

```diff
@@ -130,6 +130,11 @@ fn trace_query(query_tracker: &QueryTracker, truncated_output: &str) {
     if let Some(error_code) = &query_tracker.error_code {
         tracing::info!(target: "task_history", error_code = %error_code, "labels");
     }
+
+    if let Some(error_message) = &query_tracker.error_message {
+        tracing::error!(target: "task_history", "{error_message}");
+    }
```

Comment:
What does this look like in the task history? does it have an associated column?

Reply:
It shows up in the `error_message` column